tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -13,7 +13,6 @@ ARCH := $(shell uname -m)
 ifeq ($(ARCH), x86_64)
 EXTRA_TARBALL=cc-cloud-hypervisor-tarball \
 	cc-tdx-kernel-tarball \
-	cc-sev-kernel-tarball \
```

Comment:
Missing `\` on line 121?

I'm also wondering if we should force the kernel build every time for `cc-sev-rootfs-initrd-tarball`. If the kernel tarball has previously been built, maybe we can just check to see if a specific directory exists. Not sure how the maintainers/stakeholders feel about this. @fidencio @wainersm @fitzthum

Reply:
@ryansavino, so with this change, kernel will not be built earlier in the test. So when we are building initrd, we can force kernel build.

Reply:
@fitzthum yes. We can skip the first build. During second build, we check if we have a cached tarball that we can download. If yes, download and use that tarball. Else, build it from scratch.

Reply:
Using a cached kernel build should be fine for the sev initrd

Reply:
Right, but calling kernel build twice becomes redundant. That is why I am removing the first kernel build call and kernel will be built during initrd build. Does that look okay to you? @Alex-Carter01

Reply:
An alternative solution, as pointed out by @ryansavino, would be to depend on a previously built directory. E.g. the following work out:

```
diff --git a/tools/packaging/kata-deploy/local-build/Makefile b/tools/packaging/kata-deploy/local-build/Makefile
index 28202e90e..1ffb82713 100644
--- a/tools/packaging/kata-deploy/local-build/Makefile
+++ b/tools/packaging/kata-deploy/local-build/Makefile
@@ -117,7 +117,9 @@ cc-qemu-tarball:
 cc-rootfs-image-tarball:
        ${MAKE} $@-build
 
-cc-sev-rootfs-initrd-tarball: cc-sev-kernel-tarball
+$(MK_DIR)/build/cc-sev-kernel/builddir: cc-sev-kernel-tarball
+
+cc-sev-rootfs-initrd-tarball: $(MK_DIR)/build/cc-sev-kernel/builddir
        ${MAKE} $@-build
 
 cc-tdx-rootfs-image-tarball:
```

The approach proposed on this PR has the caveats which are:
a) If the kernel was built already for some obscure reason (e.g. the developer built it manually), it will still be built twice:
```
make cc-sev-kernel-tarball && make cc-sev-rootfs-initrd-tarball
```
b) The sev kernel is included in the cc tarball only and only if the sev initrd is built, it might be error prone

And the caveats of depending on the build directory as I showed above:

a) It doesn't depend on a specific kernel version so that the build directory may exist but the initrd requires a newer kernel version, therefore the initrd build will fail (I really hope it fails!)

In the end it is a matter of preference. I'm okay to stick with the approach proposed on this PR. I just wanted to raise the alternative to keep you confused :P

Reply:
@wainersm my thoughts about the caveats of the approach suggested in this PR:
- Only kernel should not be built on these CI systems for any obscure reason. If someone wants to use AMD SEV, they need the kernel and the initrd. Thus, I can't think of any scenario where only AMD SEV kernel needs to be built. Thus, if someone wants to use SEV, they can just call the initrd build and we will have two binaries.
On the approach you suggested, I think I agree with you. 
Personally, I prefer the approach suggested in this PR.